libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -1,6 +1,5 @@
 function(_get_common_test_compile_options output_var c_test flags)
   _get_compile_options_from_flags(compile_flags ${flags})
-
```

Comment:
add this newline back such that there's no hunk here, and then we're good to go here.

Reply:
Good catch, fixed